src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -1421,6 +1425,41 @@ public ImportFormatPreferences getImportFormatPreferences() {
                 isKeywordSyncEnabled());
     }
 
+    public static SavePreferences loadForExportFromPreferences(JabRefPreferences preferences) {
```

Comment:
Convert these two methods to instance methods and rename them to something like `loadSavePreferencesForExport`

Reply:
Seems, I was too tired at JabCon to see the obvious.